sales/invoicing/down_payment.rst

```diff
@@ -2,44 +2,104 @@
 Request a down payment
 ======================
 
-A down payment is an initial, partial payment, with the agreement that
-the rest will be paid later. For expensive orders or projects, it is a
-way to protect yourself and make sure your customer is serious.
+A down payment is a partial payment made by the buyer when a sales contract is concluded. This
+implies a significant commitment from both parties (seller and buyer) who must honor the contract.
+Indeed, the buyer shows his will to acquire the product by the down payment and agrees to pay the
+rest later, while the seller is obliged to provide the goods by accepting it.
+
+Sometimes a down payment is required for expensive orders or projects. That way, you can protect
+yourself and make sure that your customer is serious.
 
 First time you request a down payment
 =====================================
 
-When you confirm a sale, you can create an invoice and select a down
-payment option. It can either be a fixed amount or a percentage of the
-total amount.
+When a sales order is confirmed, you will then have the possibility to create an invoice. Invoices
+will be automatically created in drafts so that you can review them before validation. To create an
+invoice, Odoo Sales will offer you 3 options:
+
+- Regular invoice
+- Down payment (percentage)
+- Down payment (fixed amount)
+
+In terms of down payment, it can either be a fixed amount or a percentage of the total amount. The
+first time you will request a down payment you will be able to put the percentage or the fixed
+amount of your choice, select an income account and taxes to precise your down payment. These
+settings will be reused for future down payments.
+
+.. image:: media/down_payment_1.png
+   :align: center
+   :class: img-thumbnail
+   :alt: How can you do a down payment on Odoo Sales?
```

Comment:
See my email about alt tags.
For future docs. Don't bother too much for this one :)